packages/driver/src/multi-domain/cypress.ts

```diff
@@ -57,7 +58,10 @@ const setup = (cypressConfig: Cypress.Config, env: Cypress.ObjectLike) => {
 
   const { state, config } = Cypress
 
-  $Commands.create(Cypress, cy, state, config)
+  // @ts-ignore
```

Comment:
lots of ts-ignores, should we start updating types in here?

Reply:
For these two in particular we might need to update the types since they don't match. Probably makes sense for `cy.isCy` and `Cypress.commands`, but wasn't sure if it was worth the effort here.